Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -645,6 +669,24 @@ open class ChatMessageContentView: _View, ThemeProvider {
     @objc open func handleTapOnDeliveryStatusView() {
         delegate?.messageContentViewDidTapOnDeliveryStatusIndicator(indexPath?())
     }
+    
+    // MARK: - UITextViewDelegate
+    
+    open func textView(_ textView: UITextView, shouldInteractWith URL: URL, in characterRange: NSRange, interaction: UITextItemInteraction) -> Bool {
+        if let mentionedUsers = content?.mentionedUsers, !mentionedUsers.isEmpty {
+            let tappedMentionedUser = textViewUserMentionsHandler.mentionedUserTapped(
```

Comment:
```suggestion
            let mentionedUser = textViewUserMentionsHandler.mentionedUserTapped(
```

Reply:
This will clash with the property below. The reason we don't use the if let here, is to make the code easier to read